src/cowboy_req.erl

```diff
@@ -15,6 +15,9 @@
 
 -module(cowboy_req).
 
+-define(INVALID_COOKIE_HEADER_ERROR, exit({response_error, invalid_header,
+    'The set-cookie header is special and must be set using cowboy_req:set_resp_cookie/3,4.'})).
```

Comment:
No macros please.

Reply:
for sure. will move to a function

Reply:
Just inline is fine. There isn't too many of them. The messages are likely to be slightly edited later on anyway depending on context.